tests/filecheck/dialects/arith/arith_cfg.mlir

```diff
@@ -0,0 +1,32 @@
+// RUN: xdsl-opt %s
```

Comment:
Could we use the test dialcet here?

Also is this really a property of arith or just a property of the ir-structure in general? If so, we could propably replace everything with the test dialect and move this to a new folder "ir-structure" or something.

Reply:
No, we can unfortunately not. This test case tests the custom parsing of the `arith.add` operation, so we need at least arith to stick around. Or are you suggesting I replace the func.func and cf.br operation with the test dialect? I guess that could be possible.

Reply:
Yes, I think the func and cf dialects could certainly go.

Ah yes, I see the changes happen within arith, so that one can't go :+1:

Reply:
The test.termop does not allow `> 0` successors. Are you suggesting we expand termop to allow this?

Reply:
Yes, that would be needed too, I guess. IMO, you can do this in this PR, I don't mind. It feels like a rather small change...

Reply:
I tried this, but the build breaks as I cannot get var_successor_def to work.

Reply:
@webmiche, can you see if you can get the test dialect fixed here?

Reply:
Fixed it. Let's see what Mathieu thinks about my fixes and then we can merge, I guess.